examples/twilio-chatbot/server.py

```diff
@@ -0,0 +1,42 @@
+import asyncio
+from fastapi import FastAPI, WebSocket, WebSocketDisconnect
+from fastapi.middleware.cors import CORSMiddleware
+import uvicorn
+from starlette.responses import HTMLResponse
+from test_bot import run_bot
+
+app = FastAPI()
+
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["*"],  # Allow all origins for testing
+    allow_credentials=True,
+    allow_methods=["*"],
+    allow_headers=["*"],
+)
+
+
+@app.post('/start_call')
+async def start_call():
+    print("POST TwiML")
+    return HTMLResponse(content=open("templates/streams.xml").read(), media_type="application/xml")
+
+
+ws_tasks = []
+
+
+@app.websocket("/ws")
+async def websocket_endpoint(websocket: WebSocket):
+    await websocket.accept()
+    print("WebSocket connection accepted")
+    loop = asyncio.get_running_loop()
+    task = loop.create_task(run_bot(websocket))
+    try:
+        await task
```

Comment:
Ideally, we don't wait to `await` for the task here. This way we can create many bots.

Reply:
@aconchillo w/o await websokets won't work in my case

Reply:
You know what... that might be something that FastAPI handles. I think you don't even need a task or the `ws_task`. Only:

```
@app.websocket("/ws")
async def websocket_endpoint(websocket: WebSocket):
    await websocket.accept()
    await run_bot(websocket)
```